### PR TITLE
Onboard E2E tests in openshift CI

### PR DESCRIPTION
**Pull request by flacatus**
For now lets run only in rhtap-installer. I will create issue to expando to all repos

### CODE REVIEW

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-rhtap-installer-main-rhtap-installer-e2e

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-rhtap-installer-main-rhtap-installer-e2e

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-rhtap-installer-main-rhtap-installer-e2e

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-rhtap-installer-main-rhtap-installer-e2e

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-rhtap-installer-main-rhtap-installer-e2e

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-rhtap-installer-main-rhtap-installer-e2e

wrong creds.. retrying

File: ci-operator/step-registry/redhat-appstudio/rhtap-installer/e2e-test/redhat-appstudio-rhtap-installer-e2e-test-commands.sh [hunk 1]
```diff
@@ -3,4 +3,32 @@ set -o errexit
...
+GITHUB_ORGANIZATION="rhtap-rhdh-qe"
+GITHUB_TOKEN=$(cat /usr/local/rhtap-ci-secrets/rhtap/gihtub_token)
+RED_HAT_DEVELOPER_HUB_URL=https://"$(oc get route developer-hub -n rhtap -o jsonpath='{.spec.host}')"
+
+cd "$(mktemp -d)"
+
+oc create namespace openshift-gitops || true
+oc label namespace openshift-gitops argocd.argoproj.io/managed-by=openshift-gitops || true
+
+git clone https://github.com/redhat-appstudio/rhtap-e2e.git .
```
Comment (xinredhat) [inline ci-operator/step-registry/redhat-appstudio/rhtap-installer/e2e-test/redhat-appstudio-rhtap-installer-e2e-test-commands.sh:30]: @flacatus could you `git clone` it into a specific directory? the currently directory probably has other directories/files.

Comment (flacatus) [reply ci-operator/step-registry/redhat-appstudio/rhtap-installer/e2e-test/redhat-appstudio-rhtap-installer-e2e-test-commands.sh:30]: no, im cloning in a temporary dir. Before cloning im doing cd in a mktemp

File: ci-operator/step-registry/redhat-appstudio/rhtap-installer/e2e-test/redhat-appstudio-rhtap-installer-e2e-test-commands.sh [hunk 2]
```diff
@@ -3,4 +3,32 @@ set -o errexit
...
+APPLICATION_ROOT_NAMESPACE="rhtap-e2e-ci"
+QUAY_IMAGE_ORG="rhtap_qe"
+GITHUB_ORGANIZATION="rhtap-rhdh-qe"
+GITHUB_TOKEN=$(cat /usr/local/rhtap-ci-secrets/rhtap/gihtub_token)
+RED_HAT_DEVELOPER_HUB_URL=https://"$(oc get route developer-hub -n rhtap -o jsonpath='{.spec.host}')"
+
+cd "$(mktemp -d)"
+
+oc create namespace openshift-gitops || true
+oc label namespace openshift-gitops argocd.argoproj.io/managed-by=openshift-gitops || true
```
Comment (xinredhat) [inline ci-operator/step-registry/redhat-appstudio/rhtap-installer/e2e-test/redhat-appstudio-rhtap-installer-e2e-test-commands.sh:28]: why does it need the above 2 commands?

Comment (flacatus) [reply ci-operator/step-registry/redhat-appstudio/rhtap-installer/e2e-test/redhat-appstudio-rhtap-installer-e2e-test-commands.sh:28]: Damn, forgot to remove them. I will do it

**Pull request merged**